### ISSUE TITLE
Lowercase namespace

### ISSUE THREAD

**Issue by M-Zuber** [octokit/octokit.net#1208]
At some point a folder called `fixtures` was introduced.
This lead to a number of classes having the namespace`Octokit.Tests.Integration.fixtures`

Will make a PR to fix once #1144 is merged (as it introduces some more files to that namespace)

([Affected Files - potentially more](https://github.com/octokit/octokit.net/search?utf8=%E2%9C%93&q=namespace+Octokit.Tests.Integration.fixtures))

**Comment by Haacked** [comment]
I would love to remove all namespaces from unit and integration tests. They really don't serve a purpose for our tests. Namespaces are useful for libraries that you reference in order to disambiguate class names. But you never reference a set of unit tests for your own code. At least, we don't need to support that if you do for some wild reason. :stuck_out_tongue:

**Comment by M-Zuber** [comment]
Would you be open to one fat PR that rips out all namespace parts lower than what matches the project? 
Or shall I split it up?

**Comment by Haacked** [comment]
Split it up. I worry that one big fat PR will cause problems for everybody else. :)

**Comment by shiftkey** [comment]
Just a heads up - removing the namespaces is likely to cause a whole bunch of indenting changes to the test suite. It's a minor nuisance but things like `git blame` will then be full of lies...

**Comment by M-Zuber** [comment]
What do you mean by `indenting changes`?

**Comment by M-Zuber** [comment]
An example of what things look like now is
![image](https://cloud.githubusercontent.com/assets/4746537/14064333/f1fd22b0-f407-11e5-849b-4b315e86fe3b.png)
@Haacked do you want to just drop the `.Clients` or drop the namespace altogether (like :arrow_down: )?
![image](https://cloud.githubusercontent.com/assets/4746537/14064338/16c428c8-f408-11e5-80ad-9bd2b3ac5549.png)

**Comment by shiftkey** [comment]
@M-Zuber that's exactly the issue, and why I kinda put it aside when I started finding these 😢

**Comment by M-Zuber** [comment]
IMO I would do the first option. having a namespace that matches the project name does not create noise (at least to me) - and that is what is included in the default.

Is there a setting that prevents VS from tacking on namespace parts to match the file system?

**Comment by shiftkey** [comment]
> Is there a setting that prevents VS from tacking on namespace parts to match the file system?

[Unfortunately it seems to be R# only](http://stackoverflow.com/a/6246679/1363815)

**Comment by M-Zuber** [comment]
I am voting for leaving a namespace that matches the project name, but dropping any further levels.

Files that currently have **no** namespace can stay that way.
I would also include adding a test to `Conventions` to the PR.

@shiftkey @Haacked @ryangribble any feedback please?

**Comment by ryangribble** [comment]
It's very unfortunate because not even thinking about files where extra namespace levels were added due to folder path, we seem to have a real mix of unit and integration tests where some files have a namespace and others have none.  My OCD wants things to be consistent, particularly now it's been brought to attention, but yeah im not sure if the code/indentation churn is worth it...

> I would also include adding a test to Conventions to the PR.

Pretty tough to do when there is inconsistency in the files.  Or are you saying the convention test would "pass" if there is no namespace OR a namespace matching project, and would only fail when a namespace was present that didn't match the project?

**Comment by M-Zuber** [comment]
Yeah I meant the the test should only fail if there **is** a namespace and it doesn't match the project

**Comment by M-Zuber** [comment]
So how do we reach a consensus on this, so I can either code or close?

**Comment by shiftkey** [comment]
@M-Zuber assigning myself to look at this tomorrow
